docs/examples/greeting.js

```diff
@@ -19,7 +19,14 @@ client.on('ready', () => {
 
 // Create an event listener for new guild members
 client.on('guildMemberAdd', member => {
-  // Send the message to a designated channel on a server:
+  // Send the message to a first channel on a server:
```

Comment:
"to a first channel on a server"
you mean, "to **the** first channel on a server"